### PR TITLE
Feature/update workspace configs

### PR DESCRIPTION
**Pull request by tam-nguyen-senseye**
* (ios) updated DEVELOPMENT_TEAM and PRODUCT_BUNDLE_IDENTIFIER
* (android) refactored example app package path: co.senseye.reactnative.sdk.example

### CODE REVIEW

File: example/ios/SenseyeSdkExample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -144,6 +144,7 @@
 				ORGANIZATIONNAME = "Senseye, Inc.";
 				TargetAttributes = {
 					13B07F861A680F5B00A75B9A = {
+						DevelopmentTeam = E93K46863C;
```
Comment (Haplo-Dragon) [inline example/ios/SenseyeSdkExample.xcodeproj/project.pbxproj:147]: Presumably this our group development team ID for all of Senseye? I remember there was some weirdness with that at the beginning.

**Review by Haplo-Dragon — APPROVED** [review]
Looks good to me! :+1:

File: example/ios/SenseyeSdkExample.xcodeproj/project.pbxproj (hunk 1 above)
Comment (bobbys-dev) [reply example/ios/SenseyeSdkExample.xcodeproj/project.pbxproj:147]: That is Senseye's ID yes. I believe outside developers will have to change this team id and use a unique bundle id to their usage regardless of what is coded here.

**Review by bobbys-dev — APPROVED** [review]
Looks good. Can you add in the instruction of example app (iOS section) that the prospective developer will need to use their own unique bundle id and developer account/team id?

**Review by ach206 — APPROVED** [review]

**Review by dkman94 — APPROVED** [review]
Looks good! @bobbys-dev and I were investigating the bundle id updates a bit. Here's an example from another SDK with a similar issue:
https://developer.dji.com/mobile-sdk/documentation/quick-start/index.html

(See Obj-C app key setup where it mentions updating the bundle identifier to use a unique value)

**Comment by tam-nguyen-senseye** [comment]
> Looks good! @bobbys-dev and I were investigating the bundle id updates a bit. Here's an example from another SDK with a similar issue:
> https://developer.dji.com/mobile-sdk/documentation/quick-start/index.html
> 
> (See Obj-C app key setup where it mentions updating the bundle identifier to use a unique value)

**Comment by tam-nguyen-senseye** [comment]
accidentally closed 😓

File: example/README.md [hunk 2]
```diff
@@ -25,24 +25,24 @@ For iOS, make sure to also install the CocoaPods dependencies:
 npx pod-install
 ```
 
-Note that the CocoaPods dependencies will _not_ be included when installing dependencies via `yarn bootstrap`, which can be ran at the root directory of this repository.
+Note that the CocoaPods dependencies will **_not_** be included when installing dependencies via `yarn bootstrap`, which can be ran at the root directory of this repository.
```
Comment (Haplo-Dragon) [inline example/README.md:28]: Minor grammar:
```suggestion
Note that the CocoaPods dependencies will **_not_** be included when installing dependencies via `yarn bootstrap`, which can be run at the root directory of this repository.
```

**Pull request merged**